versioned_docs/version-v1.1/advanced/storagenetwork.md

```diff
@@ -52,8 +53,12 @@ kubectl apply -f https://raw.githubusercontent.com/harvester/harvester/v1.1.0/de
 	- Please refer Networking page for more details and configure `Cluster Network` and `VLAN Config` but not `Networks`.
 - IP range for Storage Network
 	- IP range should not conflict or overlap with Kubernetes cluster networks(`10.42.0.0/16`, `10.43.0.0/16`, `10.52.0.0/16` and `10.53.0.0/16` are reserved).
-	- IP range should be IPv4 CIDR format and 4 times the number of your cluster nodes. Longhorn will use 2 IPs per node, and the upgrade process will run two versions of Longhorn simultaneously. It will consume 4 IPs per node in the upgrade process.
-	- If your cluster has 250 nodes, the IP range should be larger than `/22`.
+	- IP range should be IPv4 CIDR format and Longhorn pods use Storage Network as following:
```

Comment:
please refer to v1.2 changes for the v1.1 doc, thanks.

Reply:
Updated per suggestion, thanks.